src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const arrayOfMen = people.filter(person => person.sex === 'm');
+  const arrayOfAges = (!century)
```

Comment:
consider using more descriptive names like `menList` and `agesList`. Writing data type in variable name is not a good approach